content/en/docs/kubernetes/collector/components.md

```diff
@@ -520,3 +524,138 @@ subjects:
     name: otel-collector-opentelemetry-collector
     namespace: default
 ```
+
+## Kubernetes Objects Receiver
+
+| Deployment Pattern   | Usable                                                   |
+| -------------------- | -------------------------------------------------------- |
+| DaemonSet (Agent)    | Yes, but will result in duplicate data                   |
+| Deployment (Gateway) | Yes, but more than one replica results in duplicate data |
+| Sidecar              | No                                                       |
+
+The kubernetes Objects receiver collects, either by pulling or watching, objects
+from the Kubernetes API server. The most common use case for this receiver is
+watching Kubernetes events, but it can be used to collect any type of Kubernetes
+object. Since the receiver gathers telemetry for the cluster as a whole, only
+one instance of the receiver is needed across the cluster in order to collect
+all the data.
```

Comment:
```suggestion
The Kubernetes Objects receiver collects, either by pulling or watching, objects
from the Kubernetes API server. The most common use case for this receiver is
watching Kubernetes events, but it can be used to collect any type of Kubernetes
object. Since the receiver gathers telemetry for the cluster as a whole, only
one instance of the receiver is needed across the cluster in order to collect
all the data.
```

Reply:
Updated